recipe-client-addon/test/browser/browser_NormandyApi.js

```diff
@@ -15,7 +15,23 @@ add_task(function* () {
   });
 
   // Test that NormandyApi can fetch from the test server.
-  const response = yield NormandyApi.get("browser/test_server.sjs");
-  const data = yield response.json();
-  Assert.deepEqual(data, {test: "data"}, "NormandyApi returned incorrect server data.");
+  let response = yield NormandyApi.get("browser/test_server.sjs");
+  let data = yield response.json();
+  Assert.deepEqual(data, {queryString: {}, body: {}}, "NormandyApi returned incorrect server data.");
```

Comment:
I suspect it will. Can you use `sjs` files in xpcshell tests? All the docs I found on them were mochitest docs.